cmd/minishift/cmd/start.go

```diff
@@ -575,6 +575,7 @@ func initStartFlags() *flag.FlagSet {
 	startFlagSet.String(configCmd.Memory.Name, constants.DefaultMemory, "Amount of RAM to allocate to the Minishift VM. Use the format <size><unit>, where unit = MB or GB.")
 	startFlagSet.String(configCmd.DiskSize.Name, constants.DefaultDiskSize, "Disk size to allocate to the Minishift VM. Use the format <size><unit>, where unit = MB or GB.")
 	startFlagSet.String(configCmd.HostOnlyCIDR.Name, "192.168.99.1/24", "The CIDR to be used for the minishift VM. (Only supported with VirtualBox driver.)")
+	startFlagSet.Bool(configCmd.SkipPreflightChecks.Name, false, "Skip the Startup checks.")
```

Comment:
I'll fix this on merge... unnecessary caps

Reply:
s/Skip the Startup checks./Skip the startup checks. Is this you want to change.